sound/soc/sof/intel/hda-sdw-bpt.c

```diff
@@ -90,6 +90,89 @@ static int chain_dma_trigger(struct snd_sof_dev *sdev, unsigned int stream_tag,
 	return sof_ipc_tx_message_no_reply(sdev->ipc, &msg, 0);
 }
 
+static struct hdac_ext_stream *
```

Comment:
Can you fix the typo with BPT in the commit message?

Reply:
fixed